perception/iphone_reader/src/iphone_reader/api.py

```diff
@@ -0,0 +1,137 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
+
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+import logging
+import threading
+from typing import Optional, Callable
+from dataclasses import dataclass
+
+import numpy as np
+from scipy.spatial.transform import Rotation as R
+
+from record3d import Record3DStream, IntrinsicMatrixCoeffs
+
+log = logging.getLogger(__name__)
+
+
+@dataclass
+class CameraMetadata:
+    intrinsic_mat: np.ndarray
+    device_type: int  # TrueDepth = 0, LiDAR = 1
+
+
+@dataclass
+class R3dFrame:
+    pose_mat: np.ndarray
+    pose_pos: np.ndarray
+    pose_quat: np.ndarray
+    color_img: Optional[np.ndarray]
+    depth_img: Optional[np.ndarray]
+
+
+class iPhoneReader:
```

Comment:
I suggest rename to Record3DReader() which is more self explaining

Reply:
I'll leave the package name as is because to a potential user without knowledge of what Record3d is, iphone_reader is more informative.
I renamed the API class name to `Record3dReader` since users reading the doc will know that they need the Record3D app and thus this name is more transparent in terms of its functionality.